packages/containers-shared/src/client/models/PortRange.ts

```diff
@@ -0,0 +1,14 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
```

Comment:
remove? (last time I comment this in this review but I think it should be removed everywhere)